.env.sample

```
@@ -51,3 +51,6 @@ AI_API_BASE=https://ai.good-job.shop # 개발
 GOOGLE_APPLICATION_CREDENTIALS="C:\Users\wkdtk\OneDrive\바탕 화면\key\long-leaf-471013-a8-f98dd975b463.json"
 GCP_BUCKET_NAME=team-ooleem
 
+#=================pynote=====================================
+HF_TOKEN=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
+PYANNOTE_MODEL=pyannote/speaker-diarization-3.1
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)